admin/admin-base.php

```diff
@@ -266,27 +266,31 @@ public function admin_print_footer_scripts() {
 		<script type="text/javascript">
 			if (typeof jQuery != 'undefined') {
 				jQuery(
-					function($){
-						$.ajaxPrefilter(function (options, originalOptions, jqXHR) {
+					function( $ ){
+						$.ajaxPrefilter( function ( options, originalOptions, jqXHR ) {
 							if ( -1 != options.url.indexOf( ajaxurl ) || -1 != ajaxurl.indexOf( options.url ) ) {
```

Comment:
Is this test still useful?  The original version used to modify `options.url` but that's not the case anymore.

Reply:
The first condition, maybe to test only URL which concerns WordPress ajax request. The second one I'm not sure to understand what it tests.